t/cobrand/bucks.t

```diff
@@ -177,13 +179,27 @@ for my $test (
     };
 }
 
+my ($report2) = $mech->create_problems_for_body(1, $body->id, 'Drainage problem', {
```

Comment:
This test block needs to be higher up (after `flytipping off road sent to extra email` I guess?), otherwise `$cobrand` has been redefined and it makes an external call to Tilma and doesn't work offline. (This is why the coverage jumped, sadly :) ).